ui/packages/catalog-ui-search/src/main/webapp/js/controllers/cesium.layerOrdering.spec.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import { expect } from 'chai'
+import { CesiumLayerOrdering } from './cesium.layerCollection.controller'
+
+describe('CesiumLayerOrdering', () => {
+  const checkOrdering = ({ actual, expected }) =>
+    expect(actual).to.have.same.ordered.members(expected)
+
+  describe('addLayer()', () => {
+    it('Can add layer to empty ordering', () => {
+      const newLayerOrder = CesiumLayerOrdering.addLayer({
+        prev: [],
+        cur: ['a', 'b', 'c', 'd', 'e'],
+        layer: 'a',
+      })
+      checkOrdering({ actual: newLayerOrder, expected: ['a'] })
+    })
```

Comment:
❓ Should we add a test with the reverse case? To ensure we fail correctly when `cur` is not a superset?